www/source/partials/docs/_glossary-artifacts.html.md.erb

```diff
@@ -0,0 +1,37 @@
+# <a name="glossary-artifacts" id="glossary-artifacts" data-magellan-target="glossary-artifacts">Artifacts</a>
+
+A Habitat artifact refers to a binary distribution for a given package built with Habitat. A Habitat artifact is a signed tarball with a `.hart` file extension. Habitat artifacts are composed of a software library or application, the configuration information for that software, and lifecycle hooks. They are created from a the plan file, a `plan.sh` on Linux systems or a `plan.ps1` on Windows systems, and are built with Habitat tools. Habitat artifacts can be exported to a specific format, such as when creating a Docker container.
+
+Habitat artifacts are identified using a four-part fully-qualified package identifier(FQPI) that takes the form: `origin/name/version/release`, where origin, name, version, and release are replaced with their corresponding values.
+
+**Package Identifier components**
+
+- `Origin`: A name that defines a set of related Habitat packages and the artifacts they generate. For example, "sample", "core", or "mycompany".
+- `Name`: The name of the application or service. For example, "postgres".
+- `Version`: The version number designation by the author(s) of the application or service. For example, "3.1.1", or "20160118".
+- `Release`: The unique Habitat id for a given version based on the timestamp pattern _YYYYMMDDhhmmss_. For example, "20160204220358" would be a Habitat artifact built at 22:03:58 on February 4th, 2016. Multiple releases of a given Habitat artifact version may exist using different dependencies and/or compiler options.
```

Comment:
```suggestion
- `Release`: The unique Habitat id for a given version based on the timestamp pattern _YYYYMMDDhhmmss_. For example, "20160204220358" would be a Habitat artifact built at 22:03:58 on February 4th, 2016.
```
The sentence "Multiple releases of a given Habitat artifact version may exist using different dependencies and/or compiler options." doesn't make much sense to me. The `release` part of the FQPI has nothing to do with dependencies or compiler options: it's purely a timestamp indicating when the package was created (according to the local clock of the machine that built it).

It *may* be worth mentioning that two different artifacts may have the same FQPI if they were built at the "same" time*. The artifacts generated should exactly the same if the plan files were identical, but they may differ otherwise. This could be the result of different plan settings, including different package targets.

\* That is, the local machine time's were identical; whether they were built at the same second in reality may or may not be true.